Content.Server.Database/Model.cs

```diff
@@ -441,6 +441,7 @@ public class Profile
         public string EyeColor { get; set; } = null!;
         public string SkinColor { get; set; } = null!;
         public int SpawnPriority { get; set; } = 0;
+        public int SquadPriority { get; set; } = 0;
```

Comment:
This should just be storing the squad's entity id then you don't need to map back and forth with an integer